api/v1beta1/foundationdb_labels.go

```diff
@@ -29,6 +29,10 @@ const (
 	// config map.
 	LastConfigMapKey = "foundationdb.org/last-applied-config-map"
 
+	// OutdatedConfigMapKey provides the annotation name we use to store the
+	// timestamp when we saw an outdated config map.
+	OutdatedConfigMapKey = "foundationdb.org/outdated-config-map-seen"
```

Comment:
I wonder if we should rather put that as `new-config-map` or. `config-map-update` and putting the timestamp there when the ConfigMap changes (then we could directly set the annotation for all Pods.

Reply:
Thinking more about it I guess we want both.

Reply:
I think it will be helpful to be able to poke the kubelet with pod updates more than once per change, in case there's a blip and things get dropped. Having a separate annotation that gets triggered when we update the desired config map may also be useful, but with the implementation for `OutdatedConfigMapKey` it only shows up for a brief duration, which means we're avoiding clutter.